can_tools/scrapers/base.py

```diff
@@ -383,37 +431,73 @@ def _normalize(self):
         df = self.normalize(data)
         return self._store_clean(df)
 
+    def _validate_time_series(self, df) -> [List[Exception]]:
+        """Do checks only applicable to time series data"""
+        issues = []
+        if (df["measurement"] == "cumulative").sum() > 0:
```

Comment:
would `(df["measurement"] == "cumulative").any()` work?

Reply:
Yep that would probably work